content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -49,34 +49,41 @@ meta:
 
 `.dvc` files can contain the following fields:
 
-- `outs` (always present): List of <abbr>output</abbr> entries that represent
-  the files or directories tracked with DVC. Typically there is only one per
-  `.dvc` file (but several can be added or combined manually).
-- `deps`: List of <abbr>dependency</abbr> entries for this stage, only present
-  when `dvc import` and `dvc import-url` are used. Typically there is only one
-  (but several can be added manually).
+- `outs` (always present): List of <abbr>output</abbr> entries (details below)
+  that represent the files or directories tracked with DVC. Typically there is
+  only one per `.dvc` file (but several can be added or combined manually).
+- `deps`: List of <abbr>dependency</abbr> entries (details below) for this
+  stage, only present when `dvc import` and `dvc import-url` are used. Typically
```

Comment:
we should start going away from using stage term in the context of .dvc files

Reply:
True. Removed the term (along with a few other small fixes) in 3e8b8829.